js/packages/web/src/components/AuctionCard/index.tsx

```diff
@@ -254,13 +255,14 @@ export const AuctionCard = ({
   const isAuctionNotStarted =
     auctionView.auction.info.state === AuctionState.Created;
 
-  //if instant sale auction bid and claimed hide buttons
+  // if instant sale auction bid and claimed hide buttons
+  const outstandingBid =
+    auctionView.myBidderPot && !auctionView.myBidRedemption;
+
   if (
-    (auctionView.isInstantSale &&
-      Number(auctionView.myBidderPot?.info.emptied) !== 0 &&
-      isAuctionManagerAuthorityNotWalletOwner &&
-      auctionView.auction.info.bidState.max.toNumber() === bids.length) ||
-    auctionView.vault.info.state === VaultState.Deactivated
+    !outstandingBid &&
+    auctionView.isInstantSale &&
+    auctionView.myBidderPot?.info.emptied
```

Comment:
This is a spot worth explicit review. Getting the button to show was a bit of a struggle.

Reply:
I think the issue here is this isAuctionManagerAuthorityNotWalletOwner is that when in cases where the auction manager is viewing the auction, you want to show the button for reclaiming the nfts. I think for instant sale auctions it's a bit strange, because for an auction manager even you kind of what to not show the buttons if they are acting as a customer until after the auction ends and THEN show the reclaim items. So maybe there is a little if inside here where IF you are the auction manager, and IF the auction is over, then show the reclaim items, as oyu would expect, even fi you the auction manager have already bought something. It's confusing.